src/VisualStudio/LiveShare/Impl/Client/RemoteLanguageServiceWorkspace.cs

```diff
@@ -537,16 +534,5 @@ private static void UpdateText(ITextBuffer textBuffer, SourceText text)
                 edit.Apply();
             }
         }
-
-        private void StartSolutionCrawler()
-        {
-            if (GlobalOptions.GetOption(SolutionCrawlerRegistrationService.EnableSolutionCrawler))
-            {
-                DiagnosticProvider.Enable(this, DiagnosticProvider.Options.Syntax);
```

Comment:
This was only enabling Syntax -- does this need a similar filter for this workspace in that other code?

Reply:
yes.  i can do that.

Reply:
done.